components/browser/engine-gecko-nightly/docs/metrics.md

```diff
@@ -18,6 +18,7 @@ The following metrics are added to the ping:
 
 | Name | Type | Description | Data reviews | Extras | Expiration |
 | --- | --- | --- | --- | --- | --- |
+| gfx.composite_time |[timing_distribution](https://mozilla.github.io/glean/book/user/metrics/timing_distribution.html) |The time taken to composite a frame. On non-webrender this is the time taken in `CompositorBridgeParent::CompositeToTarget()`. On webrender, this is the time taken from the start of `WebRenderBridgeParent::CompositeToTarget()`, until the render thread has rendered the frame (in `RenderThread::HandleFrameOneDoc()`).  |[1](https://bugzilla.mozilla.org/show_bug.cgi?id=1580129#c7)||never |
```

Comment:
Aren't there a whole slew of other metrics we are "EXTRACTing" that are defined in a metrics.yaml in m-c?  Not a blocker on this PR, but why is this one called out specially?

Reply:
Heheh, *not yet* :) That's the only one that has landed :)